SCOPE.md

```diff
@@ -0,0 +1,16 @@
+# Scope
+
+Specifies procedures for constructing and conveying artifact dependency graphs (ADGs), and other related
+data structures for artifacts.  Including but not limited to:
+
+- formats for artifact identifiers
+- formats for specifying graph relationships between artifacts
+- manner of embedding identifiers for ADGs, and other related
+data structures in artifacts of various types
+- guidance on metadata which references ADGs, and other related data structures
+- guidance for build tools for:
```

Comment:
Given my suggestion of "build process" above, it's probably worth also talking about the relationship between "build tool" and "build action". Something like the former is a specific configuration of a tool on a set of inputs, and the latter is an invocation of a tool in that configuration.